### PR TITLE
Add marshmallow

### PR DESCRIPTION
**Pull request by jssuzanne**

### CODE REVIEW

**Comment by petrus-v** [comment]
I'm wondering is it's relevant to add a test that use marshmallow and test it by travis ? it would helps users to find an example how to integrate marshmallow in anyblok project!

**Comment by jssuzanne** [comment]
Another package will give an example with marshmallow.

File: README.rst [hunk 1]
```diff
@@ -33,6 +33,7 @@ You will be prompt with questions to set configuration values.
   :db_driver_name: "postgresql", the database driver name, AnyBlok use SqlAlchemy
   :db_name: "project_name", the database name
   :http_server: ["no", "anyblok_pyramid", "anyblok_pyramid+beaker", "anyblok_pyramid+gunicorn", "anyblok_pyramid+beaker+gunicorn"], anyblok_pyramid will add a pyramid http server
+  :has_anyblok_marshmallow: ["no", "yes"], add the (de)serializator anyblok_marshmallow
```
Comment (petrus-v) [inline README.rst:36]: I would call this option **anyblok_marshmallow** (without **has_**), what do you think?

File: {{cookiecutter.project_slug}}/setup.py [hunk 2]
```diff
@@ -32,6 +32,9 @@
     'gunicorn',
     {%- endif %}
     {%- endif %}
+    {%- if cookiecutter.has_anyblok_marshmallow == 'yes' %}
```
Comment (petrus-v) [inline {{cookiecutter.project_slug}}/setup.py:35]: Could be nice to not be casse sensitive on yes/no answers, I was wondering about [Internationalization](https://docs.python.org/3.5/library/locale.html#locale.YESEXPR) or using [dist.utils](https://docs.python.org/3.6/distutils/apiref.html?highlight=distutils.util#distutils.util.strtobool) but I'm a bit worried about create weakness ...

Anyway looks good!

Comment (jssuzanne) [reply {{cookiecutter.project_slug}}/setup.py:35]: or add_anyblok_marshmallow

Comment (jssuzanne) [reply {{cookiecutter.project_slug}}/setup.py:35]: +1 for Internalization

**Pull request closed without merging**